redis/src/main/scala/zio/redis/package.scala

```diff
@@ -39,6 +39,7 @@ package object redis
     with options.Scripting {
 
   type Id[+A]        = A
+  type Redis         = Has[Redis.Service]
```

Comment:
Let's start moving towards the new module encoding. This assumes the following:
- remove the type aliases like this
- extract the `Service`s to top-level traits
- propagate

Reply:
Hope, I did it as you mean)